Makefile

```diff
@@ -396,6 +397,17 @@ docker:
 		--build-arg GIT_BRANCH=${TIKV_BUILD_GIT_BRANCH} \
 		.
 
+docker_test:
+	docker build -f Dockerfile.test \
+		-t ${DEV_DOCKER_IMAGE_NAME}:${DOCKER_IMAGE_TAG} \
+		--build-arg GIT_HASH=${TIKV_BUILD_GIT_HASH} \
+		--build-arg GIT_TAG=${TIKV_BUILD_GIT_TAG} \
+		--build-arg GIT_BRANCH=${TIKV_BUILD_GIT_BRANCH} \
```

Comment:
These args are not used.

Reply:
It's used below:
ARG GIT_FALLBACK="Unknown (no git or not git repo)"
ARG GIT_HASH=${GIT_FALLBACK}
ARG GIT_TAG=${GIT_FALLBACK}
ARG GIT_BRANCH=${GIT_FALLBACK}
ENV TIKV_BUILD_GIT_HASH=${GIT_HASH}
ENV TIKV_BUILD_GIT_TAG=${GIT_TAG}
ENV TIKV_BUILD_GIT_BRANCH=${GIT_BRANCH}

Reply:
> These args are not used.

It's used below:
ARG GIT_FALLBACK="Unknown (no git or not git repo)"
ARG GIT_HASH=${GIT_FALLBACK}
ARG GIT_TAG=${GIT_FALLBACK}
ARG GIT_BRANCH=${GIT_FALLBACK}
ENV TIKV_BUILD_GIT_HASH=${GIT_HASH}
ENV TIKV_BUILD_GIT_TAG=${GIT_TAG}
ENV TIKV_BUILD_GIT_BRANCH=${GIT_BRANCH}

Reply:
In Dockerfile we needed them because we didn't copy the `.git` folder into the container. In your use case the entire git repo is mounted to the container, so it doesn't make sense to overwrite the git hash outside.

Reply:
> In Dockerfile we needed them because we didn't copy the `.git` folder into the container. In your use case the entire git repo is mounted to the container, so it doesn't make sense to overwrite the git hash outside.

OK. Good to know the background.